src/views/Manager/index.jsx

```diff
@@ -145,7 +145,7 @@ const Manager = () => {
             <div className="section-container">
               <h2 className="secondary-title">PROPERTIES</h2>
               <div className="manager__properties__container">
-                {managerData.properties.map((property) => (
+                {managerData.properties?.map((property) => (
```

Comment:
What does this `?` mark do?

Reply:
It's an optional chain marker, meaning if managerData.properties is "nullish", it will short circuit and will just return undefined instead of throwing an error. Since this is in the render method, it falls back to rendering nothing

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
Nice! I didn't know Javascript had that. Ruby has something similar called safe navigation using the `&` character.